adobe/spectrum-css
# feat(avatar): migrate s2 avatar

## Description

For `CSS-1031`.

Adds support for new avatar-size tokens (avatar-size-800 to avatar-size-1500). Updates avatar component story to support new sizes. Adds support for avatar-border-color and avatar-border-thickness.

## Validation steps

1. Access the Storybook URL for the PR (or fetch the branch and run it locally)
2. Navigate to the avatar story
3. Toggle through the new sizes added to the avatar story, verify they behave as expected and align with the added avatar sizes.

## How and where has this been tested?

Locally in Storybook.

### Regression testing

Validate:

1. The documentation pages for at least two other components are still loading, including:


4. If components have been modified, VRTs have been run on this branch:


## Screenshots

<img width="471" alt="Screenshot 2024-10-30 at 12 40 27 PM" src="https://github.com/user-attachments/assets/a12029cb-4434-4fb7-95ee-db8096a3fe8c">
<img width="501" alt="Screenshot 2024-10-30 at 12 40 21 PM" src="https://github.com/user-attachments/assets/064f4049-8a25-4813-adbd-8bf33645d33f">

## To-do list
